examples/manifold/plot_t_sne_perplexity.py

```diff
@@ -71,8 +71,11 @@
 # Another example using s-curve
 X, color = datasets.samples_generator.make_s_curve(n_samples, random_state=0)
 
+# viridis does not exist in matplotlib < 1.5
+cmap = getattr(plt.cm, 'viridis', plt.cm.YlGnBu_r)
+
 ax = subplots[1][0]
-ax.scatter(X[:, 0], X[:, 2], c=color, cmap=plt.cm.viridis)
```

Comment:
this doesn't do anything on >=2.0, does it? It only does something on err 1.5? Not sure if it's actually worth specifying?
We're also using viridis in many other places because it's the default.

Reply:
> this doesn't do anything on >=2.0, does it? It only does something on err 1.5? Not sure if it's actually worth specifying?
We're also using viridis in many other places because it's the default.

Fair point, to be honest I was aiming at minimal changes that get the examples running without error on our minimal dependencies. If we don't specify cmap, it will be jet for matlotlib <= 1.5 and I know a few peole this may make very sad ;-).